ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml

```diff
@@ -65,6 +65,74 @@
                         <jcr:content jcr:primaryType="nt:unstructured"/>
                     </policy_165205831103087>
                 </container>
+                <text jcr:primaryType="nt:unstructured">
+                    <policy_1594215389761
+                            cq:styleDefaultClasses="cmp-form input"
```

Comment:
@DeepaliAgrwl : pl check the policy class name here.

Reply:
updated the class name in the policy. The css for ths is present inside form text component clientlib